### PR TITLE
[Documentation] PR Template

### PR DESCRIPTION
**Pull request by caleybrock**
This is a template that will show up each time we make a PR in the description. It makes it easy to fill out and has a checklist of things so we don't forget anything. We can always update it however we want in the future if we are missing something or don't find other parts useful.

### CODE REVIEW

**Comment by fnormand01** [comment]
I really like the idea. In this way, all our PRs will follow the same structure. It will also be faster to create a complete description without missing any details.

**Comment by fnormand01** [comment]
However, I think we should meet and discuss what we want and what we really need in our PR's templates.

**Comment by caleybrock** [comment]
Sounds like a plan - let's chat about it after standup tomorrow.

File: docs/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context.
+
+## Type of change
+
+Please delete options that are not relevant.
+
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
```
Comment (MichaelCherry) [inline docs/PULL_REQUEST_TEMPLATE.md]: I'm a little confused on why we would be merging in something that would be classified as this? But, if we were too, it makes sense to have its own category

**Comment by caleybrock** [comment]
Michael is fixing the Synk issue in another PR.

**Pull request merged**